.github/workflows/node-test.yml

```diff
@@ -4,38 +4,24 @@ on:
   pull_request:
   push:
     branches:
-      - master
-      - stable*
+      - '*'
 
 jobs:
-  build:
+  test:
     runs-on: ubuntu-latest
-
-    name: test
+    strategy:
+      matrix:
+        node: [ 14, 16, 18 ]
```

Comment:
Since we run a matrix, could you add a summary check so it's easier to maintain a branch protection?

https://github.com/nextcloud/.github/blob/3a0592a45b5d58b558eca89f414c0933c09e0044/workflow-templates/lint-php.yml#L38-L49

Reply:
Done 👍🏼 
After this merge, looks good add eslint and prettier to run keep the code style consistent